assets/stylesheets/bootstrap/mixins/_responsive-visibility.scss

```diff
@@ -8,6 +8,7 @@
     display: block !important;
```

Comment:
I'm not sure I understand your comment. display: block was from Bootstrap, I simply added the display type of a col element following the example of the table, tr, th and td element.